misk-aws-dynamodb/src/main/kotlin/misk/dynamodb/DynamoDbHealthCheck.kt

```diff
@@ -0,0 +1,30 @@
+package misk.dynamodb
+
+import com.amazonaws.services.dynamodbv2.AmazonDynamoDB
+import javax.inject.Inject
+import javax.inject.Singleton
+import misk.healthchecks.HealthCheck
+import misk.healthchecks.HealthStatus
+import wisp.logging.getLogger
+
+@Singleton
+class DynamoDbHealthCheck @Inject constructor(
+  private val dynamoDb: AmazonDynamoDB,
+  private val requiredTables: List<RequiredDynamoDbTable>,
+) : HealthCheck {
+  override fun status(): HealthStatus {
+    for (table in requiredTables) {
```

Comment:
nit:  should we require at least one table? I assume this list could, in theory, be empty.